terminus/models/ground_plane.py

```diff
@@ -7,6 +9,12 @@ def __init__(self, size, origin, name=None):
         self.size = size
         self.origin = origin
 
+    def bounding_box(self):
+        box_origin = Point(-self.size / 2, -self.size / 2)
```

Comment:
Beware that you are doing integer division here (see e.g. http://stackoverflow.com/questions/2958684/python-division). You should do something along the lines of `x / 2.0`.